libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -470,54 +366,61 @@ - (NSString*)uniqueUserAccountIdentifier {
     return identifier;
 }
 
+-(SFOAuthCredentials *)oauthCredentials {
+
+    NSString *oauthClientId = [self oauthClientId];
+    NSString *identifier = [self uniqueUserAccountIdentifier];
+
+    SFOAuthCredentials *creds = [[SFOAuthCredentials alloc] initWithIdentifier:identifier clientId:oauthClientId encrypted:YES];
+    creds.redirectUri = self.oauthCompletionUrl;
+    creds.domain = self.loginHost;
+    creds.accessToken = nil;
+    creds.redirectUri = self.oauthCompletionUrl;
+    creds.clientId = self.oauthClientId;
+    return creds;
+}
+
 - (SFUserAccount*)createUserAccount {
```

Comment:
The only place that this is now used is in test contexts, where those contexts could be re-worked to simply create the credentials before creating the user, then using `createUserAccountWithCredentials:`.  I think we should do away with this method, commit entirely to the "account has credentials" paradigm.